source/jormungandr/tests/places_tests.py

```diff
@@ -499,6 +499,16 @@ def test_places_stop_areas(self):
         assert status == 200
         assert "places" not in response
 
+        # Search with 'Gare de'
```

Comment:
(Same than https://github.com/hove-io/navitia/pull/3912/files#diff-4b14f66a73ac88395fe8b96e60ad37d4b9c9cb059ec4271e165be3bbb41da66aR507): comment outdated (de/du)
```suggestion
        # Search with upper and lower case shouldn't impact result
```

Reply:
:+1: